docs/merkletree/treetex/main.go

```diff
@@ -312,12 +321,47 @@ func modifyRangeNodeInfo() error {
 	return nil
 }
 
+var nodeFormats = map[string]nodeTextFunc{
```

Comment:
I feel like the whole formatting thing could be factored out. There is probably a bigger issue with the `main.go` file containing too much stuff already.

Reply:
That's probably true, but maybe for another time...